apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -263,7 +270,7 @@ export const CssValueInput = ({
 
   const suffix =
     value.type === "keyword" ? (
-      <IconButton {...getToggleButtonProps()}>
+      <IconButton css={{ width: "auto" }} {...getToggleButtonProps()}>
```

Comment:
what is auto solving here?

Reply:
I found a solution for all known icon inside textfield problems: check this out, lets wait until I merge and we can rebase this one ad4e57a83e28bea3538edd057e9fef9c724dfb7e

Reply:
should be able to remove width:auto with the updates in main